### ISSUE TITLE
question about LLM and data

### ISSUE THREAD

**Issue by JiuhaiChen** [ByungKwanLee/MoAI#11]
It is a great job! I wonder the reason using InternLM-7B instead of LLama based model? You use the InternLM-7B or InternLM-chat-7B ?

And for the training data, you only use the instruction tuning data like LLaVA-Instruct-665K? Any other data for pre-training data is used? like [558K subset of the LAION-CC-SBU dataset](https://huggingface.co/datasets/liuhaotian/LLaVA-Pretrain)?

**Comment by ByungKwanLee** [comment]
InternLM is also instruction tuned model with 1.6T multilingual dataset so that it takes an advantage of adaptive language understanding more than Vicuna model. Therefore, it surely outperforms on different languages over english solely. Used InternLM-7B. We used LLaVA-Instruct-665K-based [ShareGPT4V](https://sharegpt4v.github.io/) which filters LLaVA dataset with more high-caption quality.
